gradle-plugins/preview-rpc/src/main/kotlin/org/jetbrains/compose/desktop/ui/tooling/preview/rpc/commands.kt

```diff
@@ -20,7 +20,9 @@ internal fun RemoteConnection.receiveAttach(
         if (type == Command.Type.ATTACH) {
             val version = args.firstOrNull()?.toIntOrNull() ?: 0
             if (PROTOCOL_VERSION != version) {
-                listener?.onIncompatibleProtocolVersions(PROTOCOL_VERSION, version)
+                listener?.onError(
```

Comment:
Shall/can we report the versions in question?

Reply:
Sure, but it's a little bit tricky at the moment, because right now a protocol version is received from a preview host, not from the Gradle plugin. So we need to pass the Gradle plugin version to the IDE, which is itself incompatible change of protocol. I need to think how to implement that change better, so lets postpone